projects/kit/components/line-clamp/line-clamp.component.ts

```diff
@@ -43,11 +57,22 @@ export class TuiLineClampComponent implements AfterViewInit {
     @tuiDefaultProp()
     content: PolymorpheusContent = '';
 
+    @Input()
+    @tuiDefaultProp()
+    showHint = this.options.showHint;
+
+    @Output()
+    readonly overflowChange: Observable<{isOverflown: boolean}> = this.isOverflown$.pipe(
```

Comment:
could you add e2e test, please?

Reply:
Unfortunately, I didn’t write e2e tests and I’m not sure what to do quickly and efficiently (